lib/rubocop/cli/command/auto_genenerate_config.rb

```diff
@@ -130,10 +131,10 @@ def existing_configuration(config_file)
         end
 
         def write_config_file(file_name, file_string, rubocop_yml_contents)
-          File.open(file_name, 'w') do |f|
-            f.write "inherit_from:#{file_string}\n"
-            f.write "\n#{rubocop_yml_contents}" if /\S/.match?(rubocop_yml_contents)
-          end
+          lines = /\S/.match?(rubocop_yml_contents) ? rubocop_yml_contents.split("\n", -1) : []
```

Comment:
For clarity and simplicity:

```suggestion
          lines = rubocop_yml_contents.blank? ? [] : rubocop_yml_contents.split("\n", -1)
```

Reply:
Thanks for the suggestion @dvandersluis. Unfortunately I see test failures locally with this suggestion (via `rspec ./spec/rubocop/cli/cli_auto_gen_config_spec.rb`). I think it's due to the nonequivalence when `rubocop_yml_contents` is `nil`. (Specifically, rubocop's `blank?` monkey-patch differs from rails by being undefined on `nil`.)

Reply:
Hm, yeah looks like that's because `rubocop_yml_contents` can be `nil` if the file doesn't exist because it's not otherwise initialized in `add_inheritance_from_auto_generated_file` 🤔